packages/PHGeometry/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ add_library(phgeom_io SHARED
 
 target_link_libraries(phgeom_io -lphool -L${ROOT_PREFIX}/lib -lGeom)
 
-target_link_libraries(phgeom -lphgeom_io -lSubsysReco)
+target_link_libraries(phgeom phgeom_io -lSubsysReco)
```

Comment:
This update makes the dependency correct.  In the past `phgeom` was sometimes compiled earlier than `phgeom_io` and failed at linking to it.